packages/core/stories/tokens/colors/colors.json

```diff
@@ -78,54 +73,28 @@
       "variable": "$c-success",
       "name": "Success Green",
       "value": "#388557"
+    },
+		{
+      "variable": "$c-focus",
+      "name": "Focus Hightlight",
+      "value": "#3e94cf"
     }
   ],
-  "primaryColors": [
+	"hierarchicalColors": [
```

Comment:
As we discussed earlier, we need a place to document the lighter color variants that are used in the system 
https://mayflower.digital.mass.gov/patternlab/?p=base-colors